server/internal/app/config.go

```diff
@@ -25,6 +25,8 @@ type Config struct {
 	SMTP         SMTPConfig
 	SendGrid     SendGridConfig
 	SignupSecret string
+	GCS          GCSConfig
+	AssetBaseURL string `default:"http://localhost:8080/assets"`
```

Comment:
```suggestion
	AssetBaseURL string
```

- In `ReadConfig` func, initialze it with a default value using config.ServerHost
- Rename `ServerHost`field in `Config` struct to `Host`

Reply:
Done!
Can you check this commit [0e3de14](https://github.com/reearth/reearth-cms/pull/130/commits/0e3de14e2331ae29a1742456afdfe4f0d37bc2c9) please, and let me know if that what you mean.